ui/pages/settings/networks-tab/index.scss

```diff
@@ -234,6 +234,11 @@
     }
   }
 
+  &__restrict-height {
+    max-height: 578px;
```

Comment:
Any reason for this value `578px` ?

Reply:
I was thinking the same.... could be nice to have some measurement from designer to get the height for modal.... considering we also have different resolutions of screen. wdyt?